tiledb/sm/serialization/query.cc

```diff
@@ -1421,9 +1421,6 @@ Status query_from_capnp(
                   curr_offset_size + fixedlen_size_to_copy;
             if (existing_buffer_size_ptr != nullptr)
               *existing_buffer_size_ptr = curr_data_size + varlen_size;
-            if (nullable && existing_validity_buffer_size_ptr != nullptr)
-              *existing_validity_buffer_size_ptr =
-                  curr_validity_size + validitylen_size;
           } else {
             // Accumulate total bytes copied (caller's responsibility to
```

Comment:
Let's trace the call path back to the caller, because I don't see any updates related to the copy_state in this file.

Reply:
@ihnorton many thanks for the review! Can you please be a bit more specific about what you said here? I fail to understand what you mean by updates related to copy_state.